spec/models/miq_expression_spec.rb

```diff
@@ -308,6 +308,49 @@
     context "relative date/time support" do
       around { |example| Timecop.freeze("2011-01-11 17:30 UTC") { example.run } }
 
+      context "given a non-UTC timezone" do
+        it "generates the SQL for a = expression with a value of 'Yesterday' for a date field" do
```

Comment:
@gtanzillo this and the following test fail with:

```
     Failure/Error: 1.day.ago.in_time_zone(tz).send("#{mode}_of_day")
     
     NoMethodError:
       undefined method `_of_day' for Tue, 11 Jan 2011 00:30:00 WIB +07:00:Time
     # ./app/models/miq_expression/relative_datetime.rb:50:in `normalize'
     # ./app/models/miq_expression.rb:1661:in `to_arel'
     # ./app/models/miq_expression.rb:580:in `to_sql'
     # ./spec/models/miq_expression_spec.rb:318:in `block (5 levels) in <top (required)>'
     # ./spec/models/miq_expression_spec.rb:309:in `block (5 levels) in <top (required)>'
     # ./spec/models/miq_expression_spec.rb:309:in `block (4 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # NoMethodError:
     #   undefined method `_of_day' for 2011-01-11 00:30:00 UTC:Time
     #   ./app/models/miq_expression/relative_datetime.rb:50:in `normalize'
```

Are relative date/times not supported with the `=/!=/EQUAL/NOT EQUAL` operators?